driverkit/Makefile

```diff
@@ -5,12 +5,16 @@ DRIVERKIT ?= $(shell which driverkit)
 CONFIGS := $(wildcard config/*/*.yaml)
 VERSIONS := $(patsubst config/%,%,$(sort $(dir $(wildcard config/*/))))
 VERSIONS := $(VERSIONS:/=)
-TARGET_DISTRO ?=
+TARGET_DISTRO ?= *
+TARGET_KERNEL ?= *
 S3_DRIVERS_BUCKET ?= "falco-distribution/driver"
 
 all: $(patsubst config_%,%,$(subst /,_,$(CONFIGS)))
 
-specific_target: $(patsubst config_%,%,$(subst /,_,$(wildcard config/*/${TARGET_DISTRO}_*)))
+specific_target: $(patsubst config_%,%,$(subst /,_,$(wildcard config/*/${TARGET_DISTRO}_${TARGET_KERNEL}-*)))
+
+count_specific_target:
```

Comment:
This was already done by me Max :)

Below, there's the `make stats` command (which can be further parametrized maybe...).

Or you can directly use the script it calls.

Eg.

```console
$ ./utils/driverstats config/2aa88dcf6243982697811df4c1b484bcbe9488a2/*.yaml
# eBPF probes     191
# kernel modules  80
```

Reply:
Thanks. I think we can remove it, honestly I used it to test the changes on `specific_target` filter, and I left it just in case :D

Reply:
Working on improviong/parametrizing `make stats`.

After that I'll consider this done and we can merge it!

Reply:
Pushed something ;)

Reply:
Love the hashmap simulation <3 Great work!